release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1723,6 +1722,82 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
+
+* Previously, installing an Azure cluster into an existing Azure Virtual Network (VNet) might have failed because the installation program created a default network security group, which allowed traffic from `0.0.0.0`. The failure occurred when the existing VNet had the following rule enabled in the tenant: `Rule: Network Security Groups shall not allow rule with 0.0.0.0/Any Source/Destination IP Addresses - Custom Deny`. With this fix, the installation program no longer creates the default network security group when installing a cluster into an existing VNet, and the installation succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-11796[*OCPBUGS-11796*])
+
+* During an installation, when the cluster status is `installing-pending-user-action` the installation won't complete until the status is resolved. Previously, if the user ran the `openshift-install agent wait-for bootstrap-complete` command there would be no indication of how to resolve the problem causing this status. With this update, the command output provides a message indicating which actions must be taken by the user to resolve the issue.
++
+As an example, the `wait-for` output when an invalid boot disk is used is now:
+[source,terminal]
+----
+"level=info msg=Cluster has hosts requiring user input
+level=debug msg=Host master-1 Expected the host to boot from disk, but it booted the installation image - please reboot and fix boot order to boot from disk QEMU_HARDDISK drive-scsi0-0-0-0 (sda, /dev/disk/by-path/pci-0000:03:00.0-scsi-0:0:0:0)
+level=debug msg=Host master-2 Expected the host to boot from disk, but it booted the installation image - please reboot and fix boot order to boot from disk QEMU_HARDDISK drive-scsi0-0-0-0 (sda, /dev/disk/by-path/pci-0000:03:00.0-scsi-0:0:0:0)
+level=info msg=cluster has stopped installing... working to recover installation"
+----
+(link:https://issues.redhat.com/browse/OCPBUGS-4998[*OCPBUGS-4998*])
+
+* Previously, the assisted-installer-controller on the installed cluster would run continuously even after the cluster had completed installation. Because assisted-service runs on the bootstrap node and not on the cloud, and because the assisted-service goes offline after the bootstrap node reboots to join the cluster, the assisted-installer-controller was unable to communicate with assisted-service to post updates and upload logs and loops. In this version, the bug fix modifies the assisted-installer-controller to check the cluster installation without using assisted-service, and to exit when the cluster installation is complete. (link:https://issues.redhat.com/browse/OCPBUGS-4240[*OCPBUGS-4240*])
```

Comment:
```suggestion
* Previously, the assisted-installer-controller on the installed cluster would run continuously even after the cluster had completed installation. Because assisted-service runs on the bootstrap node and not on the cloud, and because the assisted-service goes offline after the bootstrap node reboots to join the cluster, the assisted-installer-controller was unable to communicate with assisted-service to post updates and upload logs and loops. Now, the assisted-installer-controller checks the cluster installation without using assisted-service, and exits when the cluster installation is complete. (link:https://issues.redhat.com/browse/OCPBUGS-4240[*OCPBUGS-4240*])
```